sandbox/grommet-app/src/pages/sticker-sheet/components/Compare.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import { Box, Grommet, Stack, ThemeContext } from 'grommet';
-import { hpe } from 'grommet-theme-hpe';
-import { current as hpeCurrent } from '../../../themes/theme';
+import { hpe as hpeLegacy } from 'grommet-theme-hpe-v5';
```

Comment:
For clarity and reducing number of names we use?

```suggestion
import { hpe as v0 } from 'grommet-theme-hpe-v5';
```

Reply:
Removed this line actually and instead presenting sticker sheet page layout in v1 theme.

Reply:
Done.